public/js/main.js

```diff
@@ -214,26 +181,46 @@ $(document).ready(function() {
     /**
      * Get Category and send ajax request to server
      */
-    $selectCate = document.getElementById("select-category");
     $('#select-category').change(function (e) {
-        //set page of paginate = 1
-        $i = 1;
         $('#select-food').empty();
-        $('#choose-food').html($('#choose-food').attr("data-text"));
-        if (window.location.href.indexOf("create") > 0) {
-            $url = window.location.href;
-        } else {
-            $url = window.location.href + "/create";
-        }
+        $url = $('#select-food').attr('data-route');
```

Comment:
This way to get URL is ok, but you can define a js variable in `resources/views/layouts/partials/script.blade.php` and call in any js file you want.